src/db/schemas/scientist.ts

```diff
@@ -0,0 +1,22 @@
+import mongoose from "../config";
+const { Schema } = mongoose;
+
+
+
+// This Schema is responsible to do validations in the DB and create the structure for the 
+// scientists documents. 
+const ScientistSchema = new Schema({
+  firstName: { type: String, required: true },
+  lastName: { type: String, required: true },
+  DOB: { type: Date, required: true },
+  nationality: { type: String, required: true },
+  cityOfBirth: { type: String, required: true },
+  fields: [{ type: String }],
```

Comment:
esse campo não é obrigatório mesmo?

```suggestion
  fields: [{ type: String, required: true }],
```

Reply:
Nao pq nao tinha pensado nele como um campo inicial. Mas faz sentido ser obrigatorio sim.